internal/resources/resource_host.go

```diff
@@ -886,7 +889,7 @@ func resourceMetalHostDelete(d *schema.ResourceData, meta interface{}) (err erro
 	// the delete operation from Terraform (since it has no reference to the resource).
 	deleteStateConf := &retry.StateChangeConf{
 		Pending: []string{
-			string(rest.HOSTSTATE_DETACHING),
```

Comment:
We still have "Detaching" defined, even if we don't use them. Should we keep this?

Reply:
I was thinking about this.  I'm going to add this back in case this tf ver is used before the quake PR is merged